py3status/modules/uptime.py

```diff
@@ -0,0 +1,51 @@
+# -*- coding: utf-8 -*-
+
+"""
+Display the system's uptime
+
+Configuration parameters:
+        - format : what will be displayed
+
+@author Alexis "Horgix" Chotard <alexis.horgix.chotard@gmail.com>
+@license BSD
+"""
+
+
+class Py3status:
+    """
+    """
+    # available configuration parameters
+    format = "{days} days, {hours} hours, {minutes} min"
+
+    def _uptime(self):
+        """Returns uptime in seconds or None, on Linux."""
+        with open('/proc/uptime', 'r') as f:
+            up = float(f.readline().split()[0])
+        return up
```

Comment:
I could also probably inline the dict construction by directly returning `{'full_text': <stuff>}`. Don't know if that would really be better like that considering that anyway the overhead should be minimal since the uptime is not something that we need to keep track every seconds or so